pandas/tests/groupby/test_frame_value_counts.py

```diff
@@ -308,66 +308,283 @@ def test_data_frame_value_counts_dropna(
     tm.assert_series_equal(result_frame_groupby, expected)
 
 
-@pytest.mark.parametrize("as_index", [False, True])
+def _test_categorical_single_grouper(
```

Comment:
name this `assert_categorical_single_grouper` or similar, this is *not* a test rather an asserter

Reply:
Done